docs/dictionary/property/currentCard.lcdoc

```diff
@@ -0,0 +1,34 @@
+Name: currentCard
+Type: property
+
+Introduced: 2.7.2
+
+Syntax: 
+get the currentCard of <stack>
+set the currentCard of <stack> to <cardName>
+
+Value: short name of a card in a stack
```

Comment:
Can you put a `The` at the beginning of this sentence please? And if possible move the `Value` section to between `Example` and `Description`.